lib/versions.ps1

```diff
@@ -5,21 +5,24 @@ function Get-LatestVersion {
         Get latest version of app
     .DESCRIPTION
         Get latest version of app from manifest
+    .PARAMETER App
+        App's name
+    .PARAMETER Bucket
+        Bucket which the app is belong to
```

Comment:
https://www.merriam-webster.com/dictionary/belong%20to
https://ell.stackexchange.com/questions/64136/may-i-use-belong-with-is-was-in-active-sentences

```suggestion
        Bucket which the app belongs to
```